src/NHibernate/Linq/Visitors/ParameterTypeLocator.cs

```diff
@@ -247,6 +255,13 @@ protected override Expression VisitSubQuery(SubQueryExpression node)
 				}
 				else
 				{
+					// In case a parameter is related to a sequence selector we will have to get the underling item type
```

Comment:
Who's you underling? ^^
```suggestion
					// In case a parameter is related to a sequence selector we will have to get the underlying item type
```